modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,30 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-4.6 channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} 4.6 is currently the only minor version with EUS.
+
+Although there is no difference between stable-4.6 and eus-4.6
+channels until {product-title} 4.6 transitions to EUS the phase, you can switch
+to the EUS channel as soon as it becomes available. When {product-title} 4.6
+transitions to the EUS phase of its lifecycle the stable-4.6 channel will
+no longer receive subsequent z-stream updates. After you upgrade a to a version that
+is exclusive to the EUS channel, that cluster will no longer be eligible for minor
+version upgrades until upgrades to the next EUS version become available. The next
+planned EUS version is to be 4.10 and the upgrade to that version will require a
+serial set of version upgrades, such as from 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
+
```

Comment:
Does it mean, only when 4.10 EUS is available, we have to upgrade a 4.6 EUS cluster to v4.7 first, and then 4.8-4.9-4.10, but not upgrade it to 4.10 directly from the EUS channel? And before that, no v4.7 will be available for a 4.6 EUS channel?

Reply:
Yes, when upgrades from 4.6 EUS to 4.10 become available you'd switch to the eus-4.10 channel and then repeatedly upgrade from 4.6 to 4.7 to 4.8 to 4.9 to 4.10. There's no planned direct 4.6 to 4.10 upgrade at this time.

Reply:
get it, thanks.